### ISSUE TITLE
Surface pro 7 stuck on logo and shutdown

### ISSUE THREAD

**Issue by congluanbui** [linux-surface/linux-surface#884]
To day I'm update my surface pro and it not boot. when press power, the surface show logo and turn off. I cannot access in to Bios or usb recovery. I tried hard power off but the same result. Pls help me

https://user-images.githubusercontent.com/17411550/182285553-f7e3e335-9c43-46ff-aad3-f9df3a479b88.MOV

**Comment by NP-chaonay** [comment]
does you using Windows to update firmware? Or you does not update the firmware but other else?

**Comment by congluanbui** [comment]
> does you using Windows to update firmware? Or you does not update the firmware but other else?

Yes. After update os. Im using ubuntu 22

**Comment by StollD** [comment]
Maybe try a two button reset (equivalent of removing the battery): https://www.thewindowsclub.com/surface-pro-two-button-shutdown

**Comment by TheButteryToast** [comment]
I'm having the same problem did you fix it?

**Comment by bscongluanbui** [comment]
> I'm having the same problem did you fix it?

Yep. You must run out the batery. You must power up several time. I use script on other pc open the cd rom to touch the power button of surface. after a bout 2 days, I fix this
